diff --git a/node_modules/eslint-find-rules/dist/lib/rule-finder.js b/node_modules/eslint-find-rules/dist/lib/rule-finder.js
index e717569..e98c18d 100644
--- a/node_modules/eslint-find-rules/dist/lib/rule-finder.js
+++ b/node_modules/eslint-find-rules/dist/lib/rule-finder.js
@@ -35,20 +35,31 @@ function _getCurrentNamesRules(config) {
 }
 
 function _normalizePluginName(name) {
-  var scopedRegex = /(@[^/]+)\/(.+)/;
-  var match = scopedRegex.exec(name);
+    const scopedRegex = /(@[^/]+)(\/(.+))?/;
+    const match = scopedRegex.exec(name);
+
+    /* istanbul ignore if: cannot test this branch in eslint <5  */
+    if (match) {
+      if (match[3]) {
+        // @scoped/name => @scope/eslint-plugin-name
+        return {
+          module: `${match[1]}/eslint-plugin-${match[3]}`,
+          prefix: `${match[1]}/${match[3]}`
+        };
+      }
+
+      // @scoped => @scope/eslint-plugin
+      return {
+        module: `${match[1]}/eslint-plugin`,
+        prefix: match[1]
+      };
+    }
 
-  if (match) {
+    // Name => eslint-plugin-name
     return {
-      module: match[1] + '/eslint-plugin-' + match[2],
-      prefix: match[2]
+      module: `eslint-plugin-${name}`,
+      prefix: name
     };
-  }
-
-  return {
-    module: 'eslint-plugin-' + name,
-    prefix: name
-  };
 }
 
 function _isDeprecated(rule) {
diff --git a/node_modules/eslint-find-rules/src/lib/rule-finder.js b/node_modules/eslint-find-rules/src/lib/rule-finder.js
index 3e84c05..b4a40b3 100644
--- a/node_modules/eslint-find-rules/src/lib/rule-finder.js
+++ b/node_modules/eslint-find-rules/src/lib/rule-finder.js
@@ -29,22 +29,32 @@ function _getConfig(configFile) {
 function _getCurrentNamesRules(config) {
   return Object.keys(config.rules);
 }
-
 function _normalizePluginName(name) {
-  const scopedRegex = /(@[^/]+)\/(.+)/;
-  const match = scopedRegex.exec(name);
+    const scopedRegex = /(@[^/]+)(\/(.+))?/;
+    const match = scopedRegex.exec(name);
+
+    /* istanbul ignore if: cannot test this branch in eslint <5  */
+    if (match) {
+      if (match[3]) {
+        // @scoped/name => @scope/eslint-plugin-name
+        return {
+          module: `${match[1]}/eslint-plugin-${match[3]}`,
+          prefix: `${match[1]}/${match[3]}`
+        };
+      }
+
+      // @scoped => @scope/eslint-plugin
+      return {
+        module: `${match[1]}/eslint-plugin`,
+        prefix: match[1]
+      };
+    }
 
-  if (match) {
+    // Name => eslint-plugin-name
     return {
-      module: `${match[1]}/eslint-plugin-${match[2]}`,
-      prefix: match[2]
+      module: `eslint-plugin-${name}`,
+      prefix: name
     };
-  }
-
-  return {
-    module: `eslint-plugin-${name}`,
-    prefix: name
-  };
 }
 
 function _isDeprecated(rule) {
